flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/DefaultSlotPoolServiceSchedulerFactory.java

```diff
@@ -147,7 +146,7 @@ public static DefaultSlotPoolServiceSchedulerFactory create(
     }
 
     public static DefaultSlotPoolServiceSchedulerFactory fromConfiguration(
-            Configuration configuration, JobType jobType) {
+            Configuration configuration, JobType jobType, boolean isDynamic) {
```

Comment:
Here I prefer to name it as `isDynamicGraph`, because this logic is not in StreamGraph/JobGraph.